carmm/analyse/GCN.py

```diff
@@ -0,0 +1,134 @@
+def coord_number(atoms, lattice='fcc', siteIndices=None):
+    a=3.6
```

Comment:
Why is this a fixed variable? Surely it should be dependent on the element/material? I think I saw previously @OscarvanVuren raised if this should be variable - I think I would define it as an input but mandate the user has to give the lattice parameter (I.e., don't give a default)?